src/libs/WixToolset.Data/Xsd/wix.xsd

```diff
@@ -7389,6 +7389,24 @@
                     </xs:documentation>
         </xs:annotation>
       </xs:attribute>
+      <xs:attribute name="Architecture">
+        <xs:annotation>
+          <xs:documentation>
+                        Set this attribute to 'always32Bit' to mark this as a 32-bit component. This attribute facilitates
+                        the installation of packages that include both 32-bit and 64-bit components. 64-bit
+                        components must not share the same GUID with a 32-bit component.
+          If attribute is default (or specified as 'default'), the architecture set by the -arch switch to candle.exe
```

Comment:
I liked your wording better about the default in the other description, this one's a little confusing.